nibe/connection/nibegw.py

```diff
@@ -267,6 +325,21 @@ def _on_raw_coil_value(self, coil_address: int, raw_value: bytes):
         logger.info(f"{coil.name}: {coil.value}")
         self._heatpump.notify_coil_update(coil)
 
+    def _on_coil_value(self, coil_address: int, value: Union[float, int, str]):
+        try:
+            coil = self._heatpump.get_coil_by_address(coil_address)
+        except CoilNotFoundException:
+            if coil_address == 65535:  # 0xffff
+                return
+            raise
+
+        if coil.mappings and isinstance(value, int):
+            value = coil.get_mapping_for(value)
```

Comment:
We need to catch `NoMappingException` or even `DecodeException` here, probably log it and return from the method without notifying about coil update.

Reply:
I added code to ignore this error. But we do not do the same for CoilNotFoundException, maybe we should just warn and ignore for those cases too in _on_raw_coil_value and _on_coil_value?

Reply:
I agree. We can warn and ignore.